### ISSUE TITLE
Is it possible to use the powershell type provider on a module defined in a file ?

### ISSUE THREAD

**Issue by kcheick** [fsprojects/FSharp.Management#61]
i am trying to to use the type provider on some existing powershell scripts without much success. 
I was wondering if that feature is supported. if so what is the syntax to do it?
Here is an example of i tried so far :

``` F#
type PS =  PowerShellProvider< @"myModule.psm1">
```

**Comment by sergey-tihon** [comment]
It is not possible know, but definitely nice to have feature.
Do you want to implement this?

**Comment by kcheick** [comment]
I gave it a stab today and made little progress but it is not really working right now. 
It turns out that ytou can pass a path to powershell module file just like i did but the problem here is that the module is being imported under the restricted execution policy which prevents the script from being executed. I was able to get around it by setting the AuthorizationManager in HostedRuntime.fs like so :

``` F#
let initState = InitialSessionState.CreateDefault()
initState.AuthorizationManager <- new Microsoft.PowerShell.PSAuthorizationManager("Microsoft.PowerShell")
```

this allows me to list the function in the module but when i call it it returns null no matter what.
Now i am stuck, any pointer on what is happening here would be appreciated.

for reference here is my test module 

``` powershell
function DoSomething{ 
    param ([string] $test)
    $test
}
export-moduleMember -function DoSomething
```

and the the testing code 

```F#
type TestProv  = PowerShellProvider< @"testModule.psm1">
TestProv.DoSomething (test="testData") // intellisense works for the method name and parameters
|> printfn "%A"
// prints : <null>
```

**Comment by kcheick** [comment]
The powershell function was missing an output type
Here is the [PR](https://github.com/fsprojects/FSharp.Management/pull/62)
``` f#
function doSomething {
    [OutputType([string])]
    param (
        [string] $test
    )
    return $test
}

export-moduleMember -function doSomething
```

**Comment by sergey-tihon** [comment]
Released in [v0.3.1](https://www.nuget.org/packages/FSharp.Management/0.3.1)
